### DISCUSSION TITLE
Assert-Message

### DISCUSSION THREAD

**Discussion by wofo99** [fedorovvl/tso_client#370] [General]
updated client to 1.5.7.0 today. At start I get a message box
![Client assert](https://github.com/fedorovvl/tso_client/assets/24462586/d32ba9e6-ed19-493b-9736-12592a10406f)
Anything I can do about this?
thx and thumbs up for the great work ^^

**Comment by PiTi2k5** [comment]
Hey there, demolish it and rebuild it but slower ;) it stuck in upgrading state. Same issue can happen with PVP Building, always do slow upgrading and wait some sparetime.

**Reply by wofo99** [reply]
hi, did not fix it. I removed the tso-folder and picked a fresh copy of the client from github, but exactly same box pops up. Also I have no idea what "slow upgrade" means ..

**Reply by wofo99** [reply]
no, has nothing to do with test server

**Comment by wofo99** [comment]
Hi.  I am still wondering about how to get rid of the above alert...

**Reply by fedorovvl** [reply]
try to remove easter warehouse

**Comment by wofo99** [comment]
I finally understood your proposal, and it solved the issue ! many thanks, i will close it :-)
